SourceS/sdl2_to_1_2_backports.h

```diff
@@ -244,6 +245,18 @@ inline void SDLBackport_PixelformatToMask(int pixelformat, Uint32 *flags, Uint32
 	}
 }
 
+// A limited implementation of `a.format` == `b.format` from SDL2.

```

Comment:
DocBlock comments are preferred for documenting functions

```suggestion
/**
 * A limited implementation of `a.format` == `b.format` from SDL2.
 */
```

Reply:
Done